flows/assets/cache.go

```diff
@@ -134,7 +134,7 @@ func readAsset(data json.RawMessage, itemType assetType) (interface{}, error) {
 	var assetReader func(data json.RawMessage) (interface{}, error)
 
 	if itemType == assetTypeLocationHierarchy {
-		assetReader = func(data json.RawMessage) (interface{}, error) { return flows.ReadLocationHierarchy(data) }
+		assetReader = func(data json.RawMessage) (interface{}, error) { return utils.ReadLocationHierarchy(data) }
```

Comment:
Location stuff moves back into utils as Location objects are no longer exposed in expressions etc - only their paths